blist.js

```diff
@@ -1,170 +1,176 @@
 // Dependencies
 const axios = require("axios")
 const express = require("express")
-const d = require("discord.js")
-var bodyParser = require('body-parser')
-
-// Constants
+const discord = require("discord.js")
 const url = "https://blist.xyz"
 
+let bodyParser = require('body-parser')
+
 class Blist {
-    constructor(client, key) {
-        if (!key) {
-            console.log("Some functions might not be useable because you did not provide an API key.")
+    constructor(client, apiKey) {
+        if (!apiKey) {
+            console.log("Some functions might not be useable because you did not provide an API key");
```

Comment:
```suggestion
            console.warn("Some functions might not be useable because you did not provide an API key");
```
Please look into using `console.warn` instead of `console.log` for this use case.